fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -404,6 +405,7 @@ public void getData(
                     apiRequest.getAsyncAfter(),
                     apiRequest.getFormat(),
                     uriInfo,
+                    apiRequest,
```

Comment:
I don't really have a good place to put this, because github stupidly insists on limiting what lines I can comment on. Right now, we're not actually using the `HttpResponseMaker` that is being injected. You'll note that on [line 397](https://github.com/yahoo/fili/pull/447/files#diff-33f581382b3b5c860a25432e57a66854R397) we are still manually building the `HttpResponseMaker`. So it doesn't matter what customers put in the `AbstractBinderFactory`, we'll still build and use the version defined in  Fili!

So, In addition to providing an injection point in the `AbstractBinderFactory`, you _also_ need to make the `HttpResponseMaker`  a constructor argument for `DataServlet`. 

Since `DataServlet` has the `@Inject` annotation, HK2 will build it for us, and as a part of that, it will build all of the constructor arguments and pass them in to the DataServlet.